pkgs/development/python-modules/pylibftdi/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, httpserver
+, libftdi1
+, libusb1
+}:
+
+buildPythonPackage rec {
+  pname = "pylibftdi";
+  version = "0.17.0";
+  pythonAtLeast = "3"; # compatibility with httpserver
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "132c44rs0f2zm22hnhg93p47aaw6llpfzavc64dcqrn8719mlqfw";
+  };
+
+  propagatedBuildInputs = [
+    httpserver
+    libftdi1
+    libusb1
+  ];
+
+  # Several modules have been renamed for python3. ReadTimeout doesn't
+  # appear to be part of pylibftdi at all...
+  postPatch = ''
+    substituteInPlace pylibftdi/examples/bit_server.py --replace \
+      "BaseHTTPServer" "http.server"
+
+    substituteInPlace pylibftdi/examples/bit_server.py --replace \
+      "cStringIO" "io"
```

Comment:
I feel like this should be upstreamed, as python2 is EOL in 6 weeks

Reply:
Here's a link to the [upstream issue](https://bitbucket.org/codedstructure/pylibftdi/issues/35/examples-use-python-2-modules).